src/styles/less/Hud.less

```diff
@@ -247,4 +253,30 @@
 // State not done for this PT21
 .completedDay {
     background-color: var(--green);
-}
\ No newline at end of file
+}
+
+
```

Comment:
This is huge. Making the app responsive. Nice work!

Reply:
Nice!!!!